src/resources/users/resources/catalog-requests/dto/items.dto.ts

```diff
@@ -0,0 +1,6 @@
+import { IsArray } from 'class-validator';
+
+export class ItemsDto {
+	@IsArray()
+	items: string[];
```

Comment:
You have already defined the catalog request item, why not decalre the item type with it before pull request